src/main/java/org/apache/sling/feature/maven/mojos/IncludeArtifactMojo.java

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
It would be good if the renaming of the class file from IncludeArtifact to IncludeArtifactMojo would be done in a follow up change. Otherwise it is not easy to see what actually changed

Reply:
Renamed it back to IncludeArtiact. There is a lot of noise in the branch - let me know if I should create a new branch and consolidate all the changes into one commit after the code is ok